src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentService.kt

```diff
@@ -125,6 +141,22 @@ class AppointmentService(
     }
   }
 
+  fun verifyPastAppointmentFields(
+    attended: Attended?,
+    notifyProbationPractitioner: Boolean?,
+    behaviourDescription: String?
+  ) {
+    if (attended == null) {
+      throw ValidationException("Missing field 'attended' must be set for past appointment")
+    }
+    if (notifyProbationPractitioner == null) {
+      throw ValidationException("Missing field 'notifyProbationPractitioner' must be set for past appointment")
+    }
+    if (behaviourDescription == null) {
```

Comment:
Can historic appointment be created for non attendance in which case behaviour and notify PP may not be applicable?